modules/virt-must-gather-options.adoc

```diff
@@ -33,67 +33,73 @@ Using too many parallel processes can cause performance issues. Increasing the m
 
 .Scripts
 
-Each script is only compatible with certain environment variable combinations.
+Each script is compatible only with certain environment variable combinations.
 
-`gather_vms_details`:: Collect VM log files, VM definitions, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster. This script is compatible with all environment variables, but you must specify a namespace if you use the `VM` variable.
+`/usr/bin/gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is compatible only with the `PROS` variable.
 
-`gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --vms_details`:: Collect VM log files, VM definitions, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster. This script is compatible with all environment variables, but you must specify a namespace if you use the `VM` variable.
 
-`gather_images`:: Collect image and image stream custom resource information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --images`:: Collect image and image stream custom resource information. This script is compatible only with the `PROS` variable.
 
 [id="usage-and-examples_{context}"]
 == Usage and examples
 
 Environment variables are optional. You can run a script by itself or with one or more compatible environment variables.
 
+To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
```

Comment:
How about instance types and preferences? Do we need to include example of that? Since this is 4.13.0 doc?

Reply:
@dbasunag 
InstanceTypes is DP in 4.13, so it cannot be documented downstream. However, it should be documented in 4.14, when it is TP. Could you please create a Jira issue?

Reply:
As Avital stated in her Jira comment, "InstanceTypes is DP in 4.13, so it cannot be documented downstream."

Reply:
https://issues.redhat.com/browse/CNV-10833 we already tech previewed in 4.13. Are you talking about GA?

Reply:
@dbasunag @aspauldi 
I created a separate issue for the instancetypes flag and assigned both of you. It should be an easy fix. CNV-21777

Reply:
We can resolve this one, since it is tracked under a separate jira.